static/src/stylesheets/module/facia-garnett/_item.scss

```diff
@@ -427,7 +427,6 @@ $block-height: 58px;
         justify-content: space-between;
     }
 
-    .fc-item__header,
```

Comment:
This change is likely to have effects beyond media cards. `fc-item__header`'s parent (`fc-item__content`) has `display: flex`, so removing the `flex` property makes the behaviour of `fc-item__header` and its siblings less predictable. Am I missing something?

Reply:
This should be OK because `.fc-item__headet` still has the `display:flex`, it's set here https://github.com/guardian/frontend/blob/1ad53937ea8d785b1006f2487f614d6f589e32e6/static/src/stylesheets/module/facia-garnett/_item.scss#L193